test/blackbox-tests/test-cases/skip_vendored_dirs.t/run.t

```diff
@@ -0,0 +1,15 @@
+The project "root_project" vendor "a". And we are using "dirs" to skip the directory "a",
+now like a directory can have more than one status, so we choose one of them following
+this priority(example of "a"): "(dirs :standard \ a)" -> "(data_only_dirs a)" -> "(vendored_dirs a)".
```

Comment:
I find this a bit easier to understand, let me know if this is the meaning that you mean.

```suggestion
The project "root_project" vendors project "a". In this case, "a" can be excluded in multiple ways, in order of evaluation:

1. "(dirs :standard \ a)"
2. "(data_only_dirs a)"
3. "(vendored_dirs a)"
```